.gitignore

```diff
@@ -11,6 +11,7 @@ domain/build
 model/build
 testing/build
 utility/build
+model/build.gradle
```

Comment:
@DeonWaju Please not add this in `.gitignore`, remove it.
You have to not stage or commit `model/build.gradle` with with your local changes.